src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -1237,6 +1250,86 @@ private void dumpGroupsRequest(SwitchId switchId, String correlationId, String t
         }
     }
 
+    private void doInstallGroupRequest(CommandMessage message) {
+        SwitchId switchId = ((InstallGroupRequest) message.getData()).getSwitchId();
+        MirrorConfig mirrorConfig = ((InstallGroupRequest) message.getData()).getMirrorConfig();
+        String correlationId = message.getCorrelationId();
+
+        final IKafkaProducerService producerService = getKafkaProducer();
+        try {
+            logger.debug("Install group '{}' for switch '{}'", mirrorConfig.getGroupId().intValue(), switchId);
+
+            context.getSwitchManager().installGroup(DatapathId.of(switchId.toLong()), mirrorConfig);
```

Comment:
You already have group manipulation commands (used by flowhs) why for you need one more set of them? You can see  how it can be used here `org.openkilda.floodlight.kafka.RecordHandler#makeFlowSegmentWrappedCommand(org.openkilda.messaging.command.flow.InstallServer42IngressFlow, org.openkilda.messaging.MessageContext, org.openkilda.floodlight.command.flow.FlowSegmentResponseFactory)`

PS You can't use just "create group" term, because you create group for mirror point, not some "generic" OF group.

Reply:
Now commands written for flowhs are used